<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterView</class>
 <widget class="QWidget" name="RegisterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/img/retouch_2025011309461031.png</normaloff>:/img/retouch_2025011309461031.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>661</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bauhaus 93</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>321</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>321</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>321</width>
     <height>60</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/d54f24a4b497ab92e85f76d924c305c.jpg);</string>
   </property>
   <widget class="QPushButton" name="homeButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/4a154c900c9b2cdcfc7e5ddb8c535cf.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>550</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: rgba(168, 90, 78, 190);
	border-radius:20px;
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1055</x>
     <y>-20</y>
     <width>250</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/dialog2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>点击左上角小房子</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文琥珀</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>可返回上一页嗷~</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>label</zorder>
  <zorder>usernameEdit</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>confirmEdit</zorder>
  <zorder>registerButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections/>
</ui>
